testdata/all-fields.out2.txt

```diff
@@ -14,7 +14,6 @@ accounts:
     - operators
     home: /home/lzap
     name: lzap
-    password: ""
```

Comment:
User password was previously not a pointer but I think it should be so I made it a pointer, this means the "not set" value is now not an empty string but nil pointer.